docs/configuration.rst

```diff
@@ -726,6 +726,17 @@ Description
     * ``true``: Original ZIP archives
     * ``false``: Converted video files
 
+extractor.danbooru.extended-metadata
+-------------------------
```

Comment:
This option should just be called `metadata`. There are already a few other options with similar functionality which are only called `metadata` as well. And the `-` underline should match the length of the entire option name.

Reply:
done

Reply:
looked at the API docs again, it might be possible to do it in one query by listing every single metadata key after the ?only,
but I'm not sure that's a good idea (will break with any future change, also didn't check if you can actually get every key like that that is returned in the default query)